### ISSUE TITLE
Footer CSS

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1493]
When using the `<p>` or `<ul>` tags in the 'Custom HTML' widget in the footer, the colors are black, but should be white.

Suggested addition to CSS (that works when I use custom CSS:

```
footer.page-footer p, footer.page-footer li {
	color: #ffffff;
```

Example:

![screen shot 2019-01-14 at 5 24 24 pm 2](https://user-images.githubusercontent.com/1569463/51151941-9ab43700-1821-11e9-8e3d-381b66e9ca3c.png)
